app/experimenter/nimbus-ui/src/components/TableVisualizationRow/significance-positive.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M18.7 10.7c-.2.2-.4.3-.7.3-.3 0-.5-.1-.7-.3L13 6.4V20c0 .6-.4 1-1 1s-1-.4-1-1V6.4l-4.3 4.3c-.4.4-1 .4-1.4 0-.4-.4-.4-1 0-1.4l6-6c.1-.1.2-.2.3-.2.2-.1.5-.1.8 0 .1.1.2.1.3.2l6 6c.4.4.4 1 0 1.4z" fill="#28A745"/></svg>
```

Comment:
This is of _little significance_ (😉), but I generally try to set SVG fills to `currentColor`, and let CSS style it (either from the `fill` property or inheriting from text `color`). That being said, I doubt we will need to change the colors of these negative/positive/neutral, just something to keep in mind. Maybe you already considered it.

Reply:
Yeah, I really briefly thought about this but figured we wouldn't be changing the color of these. If we need to down the line, we can adjust them then.